pom.xml

```diff
@@ -67,7 +67,7 @@
         <google.re2j>1.7</google.re2j>
 
         <!-- temporary version waiting for real automatic release in ecocode repository -->
-        <ecocode-rules-specifications.version>1.4.6</ecocode-rules-specifications.version>
+        <ecocode-rules-specifications.version>1.5.0-SNAPSHOT</ecocode-rules-specifications.version>
```

Comment:
this SNAPSHOT version is OK for development but we have to put here the released version 1.5.0 after making the new realese in "ecocode" repository and pushing it to maven central (with another process that we can find in documentation)

Reply:
Ok, apply